src/main/java/io/xpring/xrpl/model/XRPTransaction.java

```diff
@@ -162,6 +162,21 @@
    */
   Optional<String> deliveredAmount();
 
+  /**
+   * A boolean indicating whether this transaction was found on a validated ledger, and not an open or closed ledger.
+   *
+   * @return A boolean indicating whether or not this transactionw as found on a validated ledger.
+   * @see "https://xrpl.org/ledgers.html#open-closed-and-validated-ledgers"
+   */
+  boolean validated();
+
+  /**
+   * The index of the ledger on which this transaction was found.
```

Comment:
I don't think this is always in a "search" context. 

WDYT of: 
```
The ledger index of the ledger in which this transaction was included, or is slated to be included. 
```

Is this field set only if validated == true? Or is it possible to get an open/closed ledger where this value could change?

Reply:
So I think the `ledger_index` field is set no matter what... this is based on reading the rippled JSON API documentation for the `tx` command, and also looking at the rippled handler code here: https://github.com/ripple/rippled/blob/97712107b71a8e2089d2e3fcef9ebf5362951110/src/ripple/rpc/handlers/Tx.cpp#L253.   So it is possible that the `ledger_index` could ultimately change if it's not being reported from a validated ledger.  

As for the comment change though, I'm not sure that it necessarily clarifies that potential confusion.... it might be easy to mistake "slated to include" as "the final ledger index WILL still be this number, even though it's not validated yet" ... so for now I'm just going to leave the comment in this simple form.

If we really wanted to make the SDK completely foolproof, maybe we would force it to only ever return validated data? Otherwise we leave some of these details up to the knowledge of users.